pages/linux/showkey.md

```diff
@@ -0,0 +1,24 @@
+# showkey
+
+> Displays the keycode of pressed keys on the keyboard, helpful for debugging keyboard-related issues and key remapping.
+> More information: <https://man7.org/linux/man-pages/man1/showkey.1.html>.
+
+- View keycodes(decimal):
+
+`sudo showkey`
+
+- Display scancodes in hexadecimal:
+
+`sudo showkey -s`
+
+- Display keycodes in decimal (default):
+
+`sudo showkey -k`
+
+- Display keycodes in ascii, decimal, and hexadecimal:
+
+`sudo showkey -a`
+
+- Exit the program:
+
+`Press Ctrl + d`
```

Comment:
```suggestion
`Ctrl + d`
```
We indicate keycaps in this way across the pages. Some examples:
- [mpv](https://tldr.inbrowser.app/pages/common/mpv)
- [cmus](https://tldr.inbrowser.app/pages/linux/cmus)